projects/packages/account-protection/src/class-validation-service.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Class used to define Validation Service.
+ *
+ * @package automattic/jetpack-account-protection
+ */
+
+namespace Automattic\Jetpack\Account_Protection;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+
+/**
+ * Class Validation_Service
+ */
+class Validation_Service {
+	/**
+	 * Check if the password is in the list of compromised/common passwords.
+	 *
+	 * @param string $password The password to check.
+	 *
+	 * @return bool True if the password is in the list of compromised/common passwords, false otherwise.
+	 */
+	public function check_weak_passwords( string $password ): bool {
```

Comment:
any thoughts on [this comment](https://github.com/Automattic/jetpack/pull/41365#discussion_r1934226573) about naming it more explicitly?